pykickstart/handlers/f29.py

```diff
@@ -85,6 +85,7 @@ class F29Handler(BaseHandler):
         "snapshot": commands.snapshot.F26_Snapshot,
         "sshpw": commands.sshpw.F24_SshPw,
         "sshkey": commands.sshkey.F22_SshKey,
+        "syspurpose" : commands.syspurpose.F29_Syspurpose,
```

Comment:
Does it make sense to have this command in Fedora? Shoudn't you use a different handler?